src/cdrizzleapi.c

```diff
@@ -182,6 +183,22 @@ tdriz(PyObject *obj UNUSED_PARAM, PyObject *args, PyObject *keywords)
     goto _exit;
   }
 
+  if (kernel == kernel_tophat) {
+      if (sprintf(warn_msg,
+            "Kernel '%s' has been deprecated and it will be removed in a future release.",
+            kernel_str) < 1) {
+          strcpy(warn_msg, "Selected kernel has been deprecated and it will be removed in a future release.");
+      }
+      PyErr_WarnEx(PyExc_DeprecationWarning, warn_msg, 1);
+  } else if (kernel == kernel_gaussian || kernel == kernel_lanczos2 || kernel == kernel_lanczos3) {
+      if (sprintf(warn_msg,
+            "Kernel '%s' is not a flux-conserving kernel.",
+            kernel_str) < 1) {
+          strcpy(warn_msg, "Selected kernel '%s' is not a flux-conserving kernel.");
```

Comment:
I haven't written C in a long time but don't you need to pass a value for the `%s` formatting.

Reply:
of corse. another 🤦‍♂️